aptos-move/framework/aptos-framework/sources/genesis.move

```diff
@@ -49,6 +49,8 @@ module aptos_framework::genesis {
         // deployed to. This will be entirely managed by on-chain governance and no entities have the key or privileges
         // to use this account.
         let (aptos_framework_account, framework_signer_cap) = account::create_aptos_framework_account();
+        let (aptos_token_account, aptos_token_signer_cap) = account::create_framework_reserved_account(@0x3);
```

Comment:
can we unify these, no need for two calls.

Reply:
I remove create_aptos_framework_account and replace it with create_framework_reserved_account

Reply:
sorry I meant two different functions not having a single function that returns a bunch of signers :)